resources/views/partials/log-in-form.blade.php

```diff
@@ -1,16 +1,24 @@
 {!! Form::open(['route' => 'login']) !!}
     {{ csrf_field() }}
 
+    @if ($errors->default->first('email'))
```

Comment:
What's the circumstance in which we have the email problem on the default error bag instead of loginForm?

Reply:
If the user tries to log in with the wrong email or password then for some reason, which I'm not entirely sure the error is returned using the default error bag. The request is still a `LoginFormRequest` but it doesn't use the custom `loginForm` error bag which is a bit strange.

Reply:
Got it--thanks!